racket/collects/file/untar.rkt

```diff
@@ -62,7 +62,7 @@
   (define checksum-bytes (read-bytes* 8 in))
   (define type-byte (integer->char (read-byte in)))
   (define type (case type-byte
-                 [(#\0) 'file]
+                 [(#\0 0) 'file]
```

Comment:
Thanks for the patch! But do you mean #\nul here? The variable `type-byte` is misnamed, since the byte has been converted to a character....

Reply:
Nice catch; fixed!

Reply:
Thanks!